src/main/scala/org/reactivestreams/Publisher.scala

```diff
@@ -0,0 +1,29 @@
+package org.reactivestreams
```

Comment:
That wouldn't be OK. What this project does is to expose these interfaces to Scala.js, which is also for code that is cross compiled. And you can't have code cross-compiled if these wouldn't use the same package. The project I've been working on does this: https://monix.io - publishing these traits in order for cross compilation to work.